pdtf-api-1.2.0.yaml

```diff
@@ -0,0 +1,158 @@
+openapi: 3.0.3
+info:
+  title: Property Data Trust Framework API
+  version: 1.2.0
+  license:
+    name: MIT
+    url: https://opensource.org/licenses/MIT
+  description: This API defines how participants in the UK Property Data Trust Framework can securely share property data, together with the verifiable provenance of that data. The API is built around a variation of OpenID Connect Verified Claims schema.
+servers:
+  - url: http://api.pdtf-example.com
+    description: Example PDTF data source
+tags:
+  - name: transactions
+    description: Transactions API
+paths:
+  /transactions:
+    post:
+      tags:
+        - transactions
+      summary: Add claims on a transaction
```

Comment:
Is this 'create a new transaction'? The path for creating claims would be `POST transactions/{id}`?

Reply:
Thinking of the process flow, I wonder if it goes:

- `GET /transactions?uprn=123456` <- returns transaction ID(s?) or 404 if no transaction exists
- `POST /transactions` <- returns a new transaction ID to then add claims to
OR
- `POST /transactions/[new-unique-uuid-generated-by-the-external-system]` to add an initial claim which must contain at least a `/propertyPack/uprn` claim

Reply:
> The path for creating claims would be POST transactions/{id}
Since `POST` usually generates an identifier for the resource, I assumed we cannot have an ID in the PATH.

But I now realize that we dont have a `PUT` / `PATCH`, so we could do

1. POST /transactions <- returns a new transaction ID to then add claims to
2. POST /transactions/[transactionId] <- Update the existing transaction with a new claim (Although `POST` now looks weird - One to discuss